### ISSUE TITLE
Question about random seeds and the AtomsDataModule

### ISSUE THREAD

**Issue by MasterLucas** [atomistic-machine-learning/schnetpack#704]
Dear SchNetPack developers,

I would like to better understand how random seeds are used during model training (and inference, if applicable). Specifically, what decision-making operations rely on random seeds? Is batch sampling during training one of them? And if so, does this still apply even when I pass an initial split file to the `AtomsDataModule`?

Additionally, I have a quick question about using the `AtomsDataModule` in active learning. I am providing a specific split file with a predefined data split, but at each generation, I need to expand the training set while keeping the validation set fixed (same size and same samples). How can I achieve this? Would setting `num_train=1.0` be sufficient or do I need to add the new indices to the split file?

Yours sincerely,

Lucas Bandeira

**Comment by stefaanhessmann** [comment]
Hi @MasterLucas,

we just use the seed functions of `torch`, so it does affect all random processes in `torch`. For further details you would need to check their documentation.

Regarding the split file and active learning: I suggest to write a custom splitting strategy ([see here](https://github.com/atomistic-machine-learning/schnetpack/blob/master/src/schnetpack/data/splitting.py)) that takes a split file as an input. This would allow you to use previous splits and add new data.

Hope this helps!

Best,
Stefaan
